### PR TITLE
Update _Colors.py

### PR DESCRIPTION
**Pull request by Gasta88**
In reference to issue #820, moved self_test into doctest format.
I'm using this as a reference: https://github.com/biopython/biopython/commit/9242f12a65fb1d6bdc82f8201531662fb63a101b

When I run the verbose output for doctests, I get no error but no method is tested. 

What am I doing wrong?

### CODE REVIEW

File: Bio/Graphics/GenomeDiagram/_Colors.py [hunk 1]
```diff
@@ -30,6 +30,31 @@
 class ColorTranslator(object):
     """ Class providing methods for translating representations of color into
     """
+
+    """
```
Comment (peterjc) [inline Bio/Graphics/GenomeDiagram/_Colors.py]: You need to remove the two triple quote lines above - right now your test is not part of the docstring, rather a self contained string in the function.

**Comment by peterjc** [comment]
Try ``run_doctest(verbose=1)`` or ``run_doctest(verbose=2)``, from memory the zero setting is silent by default?

You will also need to add this doctest to the list in ``Tests/run_tests.py`` so that it will be included in the test-suite as run on TravisCI, via ``python run_tests.py`` or specifically via ``python run_tests.py doctest``.

**Comment by Gasta88** [comment]
I've added the change of Tests/run_tests.py in the PR #1092 , I couldn't merge the branch into this one from my repo.

Is the failure in codecode caused by the Verbose=2 or by the fact that the commit happened before I edited Tests/run_tests.py?

**Comment by peterjc** [comment]
I don't know why CodeCov is unhappy.

Regarding multiple pull requests - are you using git at the command line, or simply editing files via GitHub?

**Comment by Gasta88** [comment]
Since I'm using different computers, on the latest commit I've used the GitHub interface. In one of them I have restrictions on what I can install on it (work computer).

I'll use git in the future.

**Pull request merged**